pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -47,11 +47,7 @@ const (
 // Cluster type based on resolution
 // For inbound (sidecar only): Cluster for each inbound endpoint port and for each service port
 func (configgen *ConfigGeneratorImpl) BuildClusters(env *model.Environment, proxy *model.Proxy, push *model.PushContext) ([]*v2.Cluster, error) {
-	clusters := make([]*v2.Cluster, 0)
-
-	services := push.Services
-
-	clusters = append(clusters, configgen.buildOutboundClusters(env, proxy, push, services)...)
```

Comment:
please undo this change.. this file is changing in another PR and the logic below is also going to change

Reply:
ok